receiver/otlpreceiver/internal/util/errors.go

```diff
@@ -0,0 +1,23 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package util // import "go.opentelemetry.io/collector/receiver/otlpreceiver/internal/util"
```

Comment:
nit: maybe rename to `package status`? IMO `util` is an antipattern but I don't have a good alternative in mind

Reply:
if we're ok sharing a name with the existing `status` package we depend on, should I call it `errors`? That was the original name I chose but didn't like that it overlapped with the `errors` package.

Reply:
`errors` works for me as well, this is internal so we can always change it later as the package evolves

Reply:
Updated